### PR TITLE
Renewed the dependencies of the gems by gemnasium server

### PR DESCRIPTION
**Pull request by Mifrill**

### CODE REVIEW

File: spec/requests/api/v1/cors_headers_spec.rb [hunk 1]
```diff
@@ -2,9 +2,9 @@
```
Comment (zharikovpro) [inline spec/requests/api/v1/cors_headers_spec.rb]: Wildcard is not supported everywhere, must use domain for compatibility: https://stackoverflow.com/questions/13146892/cors-access-control-allow-headers-wildcard-being-ignored

**Review by zharikovpro — CHANGES_REQUESTED** [review]
Do not use wildcard CORS

**Comment by Mifrill** [comment]
Ok, but when we use new version of rack-cors gem (1.0.0), against (0.4.1) we have error:
```
     Failure/Error: expect(response.headers['Access-Control-Allow-Origin']).to eq('http://super-test.com')
     
       expected: "http://super-test.com"
            got: "*"
```
Apparently a new release was released without taking this factor
```
    # https://github.com/cyu/rack-cors
    config.middleware.insert_before 0, Rack::Cors do
      allow do
        origins '*'
        resource '*', :headers => :any, :methods => [:get, :post, :options]
      end
    end
```

**Comment by zharikovpro** [comment]
> Ok, but when we used new version of rack-cors gem (1.0.0), aganst (0.4.1) we have err

Then don't use it

**Pull request merged**